handlers/guest_links_test.go

```diff
@@ -346,3 +345,193 @@ func TestDeleteInvalidGuestLink(t *testing.T) {
 		t.Fatalf("DELETE returned wrong status code: got %v want %v", status, http.StatusBadRequest)
 	}
 }
+
+func TestGuestLinkToggleInValidRequest(t *testing.T) {
```

Comment:
The names seem to be flipped because these other test is called "TestGuestLinkOperations" but that's testing invalid input.